tests/dragonfly/cluster_test.py

```diff
@@ -3118,3 +3118,44 @@ async def test_cluster_sharded_pub_sub_migration(df_factory: DflyInstanceFactory
     assert message == {"type": "subscribe", "pattern": None, "channel": b"kostas", "data": 1}
     message = consumer.get_sharded_message(target_node=node_a)
     assert message == {"type": "unsubscribe", "pattern": None, "channel": b"kostas", "data": 0}
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_readonly_replication(
+    df_factory: DflyInstanceFactory, df_seeder_factory: DflySeederFactory
+):
+    # create cluster master and replica
+    # For now replica always should work in read-only mode
+    # READONLY command returns always OK without any impact
+    # In the future we may decide to implement the same behavior as REDIS
```

Comment:
please open an issue to actually implement this behavior. it can be minor for now.